tests/d3d12_crosstest.h

```diff
@@ -730,6 +730,15 @@ static inline bool is_amd_vulkan_device(ID3D12Device *device)
             properties.driverID == VK_DRIVER_ID_AMD_OPEN_SOURCE_KHR ||
             properties.driverID == VK_DRIVER_ID_AMD_PROPRIETARY;
 }
+
+static inline bool is_adreno_device(ID3D12Device *device)
```

Comment:
There's probably a different ifdef where you need to declare this too. It's failing CI.